Python/bltinmodule.c

```diff
@@ -2815,6 +2815,19 @@ builtin_enableinvariant_impl(PyObject *module)
     return Py_EnableInvariant();
 }
 
+/*[clinic input]
+is_pyrona_program as builtin_is_pyrona_program
+
+Returns True if the program has used regions or cowns.
+[clinic start generated code]*/
+
+static PyObject *
+builtin_is_pyrona_program_impl(PyObject *module)
+/*[clinic end generated code: output=2b6729469da00221 input=d36655a3dc34a881]*/
+{
+    return Py_is_invariant_enabled() ? Py_True : Py_False;
```

Comment:
Does feel like a bit of abuse of notions.  I think putting a TODO here, se that we don't merge phase 3 without revisiting.  Really I think we want a has used Pyrona, and that is the name of the flag that is used to determine if we should do the region check.

Reply:
I fully agree. Will add the TODO as you suggest.